Makefile

```diff
@@ -32,3 +32,13 @@ check-fix:      ## Check if the GAMS code follows the coding etiquette
                 ## and offer fixing any problems directly if possible
                 ## using gms::codeCheck
 	Rscript -e 'invisible(gms::codeCheck(strict = TRUE, interactive = TRUE))'
+
+test:           ## Test if the model compiles and runs without running a full
+                ## scenario. Tests take about 10 minutes to run.
+	$(info Tests take about 10 minutes to run, please be patient)
```

Comment:
For me, was 16 minutes actually:
```
✔ | F W S  OK | Context
✔ |         1 | 0-compile [183.7s]
✔ |         1 | 1-quick [444.6s]
✔ |         1 | 2-gamscompile [363.1s]
✔ |     1   0 | 3-default
```

Reply:
On the login nodes? Because 180 s for a compilation is pretty slow, laptops should be faster. If it is on the login nodes, then there is slurm queuing and it is hard to predict how long it will take.

Reply:
Yes, on the login nodes…